.github/workflows/pull-request-target.yml

```
@@ -0,0 +1,81 @@
+
+            1. PRs with issues not assigned to you will be closed by the reviewer
+            1. Fix the first comment in the PR so that each issue listed automatically closes
+
+            ### Other
+
+            1. :dart: Please be considerate of our volunteers' time. Contacting the person who assigned the reviewers is not advised unless they ask for your input. Do not @ the person who did the assignment otherwise.
+            2. Read the CONTRIBUTING.md file make 
+
+      - name: Greeting Message to User
+        uses: actions/first-interaction@4a1f5866f0b9dcb02a5d9640a1d9b6a0f2203eef
+        permissions:
```

actionlint (1.7.4)

unexpected key "permissions" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)